sunpy/net/base_client.py

```diff
@@ -102,7 +104,69 @@ def show(self, *cols):
         table = self.build_table()
         if len(cols) == 0:
             return table
-        return table[list(cols)]
+        tablecols = table.columns
+        valid_cols = [col for col in cols if col in tablecols]
+        return table[valid_cols]
+
+
+class BaseQueryResponseTable(BaseQueryResponse):
```

Comment:
Looks like this class is missing tests or did some builds fail?

Reply:
There are not direct tests for this that I can find.

Reply:
Want me to add some? I think we have some of it covered by client tests.

Reply:
Ah, I see. If we have some client tests then we should delegate those test cases there instead...